xml/public-cloud-setup.xml

```diff
@@ -0,0 +1,565 @@
+    API. In Microsoft Azure, all security credentials will be
+    collected during setup. In Google Compute Engine, the instance
+    must be launched with an IAM role including
+    <emphasis role="literal">Compute Admin</emphasis> and
+    <emphasis role="literal">Service Account
+    Actor</emphasis> scopes.</para>
+    <para>Select an instance size for the Administrative node that
+    meets the system requirements as documented in the
+    &amp;productname; documentation.
+    <variablelist>
+      <varlistentry>
+        <term>Memory</term>
+        <listitem>
+          <para>Minimal main memory: 8GB</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Volume Size</term>
+        <listitem>
+          <para>Root volume size should be at least 40GB and is
+          dependent on the size and number of different containers
+          running in the cluster. The default root volume size of
+          the images is smaller than 40GB in some Public Cloud
+          frameworks; you must resize the root volume to meet your
+          needs upon instance launch.</para>
+        </listitem>
+      </varlistentry>
+    </variablelist></para>
+  </sect1>
+  <sect1 xml:id="network-considerations">
+    <title>Network Considerations</title>
+    <para>&amp;productname; expects that DNS resolution is
+    functional. In general, Public Cloud providers provide a DNS
+    service that works with internal host names. You can also set
+    up your own DNS resolution; please refer to the documentation
+    of the Public Cloud provider of your choice to configure your
+    own DNS service.</para>
+    <para>It is recommended that &amp;productname; is setup to run
+    in two subnets in one network segment, also referred to as VPC
+    or VNET. The Administrative node should run in a subnet that is
+    not accessible to the outside world and should be connected to
+    your network via VPN or other means. Consider a security
+    group/firewall that only allows ingress traffic on ports 22
+    (SSH) and 443 (https) for the Administrative node from outside
+    the VPC. All nodes must have access to the Internet through
+    some route in order to connect to SCC and receive updates, or
+    be otherwise configured to receive updates (e.g. via an SMT
+    server).</para>
+    <para>Depending on the applications running in your cluster you
+    may consider exposing the subnet for the cluster nodes to the
+    outside world. Use a security group/firewall that only allows
+    incoming traffic on ports served by your workload(s). For
+    example, a containerized application providing the backend for
+    REST based services with content served over https should only
+    allow ingress traffic on port 443.</para>
+    <para>Traffic between the two subnets should be allowed to flow
+    freely. &amp;productname; uses the following ports:</para>
+    <para>
+      <emphasis role="bold">Administrative node</emphasis>
```

Comment:
+1 for initial SG instructions

Not sure how you would format it, but it would be another +1 to specify whether each rule is inbound or outbound, and whether its internal or Internet facing.